packages/mobile/ios/AudiusReactNative/Info.plist

```diff
@@ -17,7 +17,10 @@
 		<key>CFBundlePackageType</key>
 		<string>APPL</string>
 		<key>CFBundleShortVersionString</key>
-		<string>1.1.62</string>
+    <!-- 
```

Comment:
I think this was a holdover from previous commits, do we want it still?

Reply:
Nah we don't need it